### PR TITLE
feat(messaging): add concepts for Messaging MTA-2127

### PR DESCRIPTION
**Pull request by RoRoJ**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Creation of concepts page for Messaging beta.

### CODE REVIEW

File: compute/messaging/concepts.mdx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+A messaging protocol defines a structured way for users / platforms / services / applications to exchange data and messages, even if normally they do not "speak the same language". Protocols also describe how messages should be processed, prioritized, managed and routed. Messaging protocols currently supported by Scaleway Messaging include [NATS](#nats), [SQS](#sqs) and [SNS](#sns).
+
+</Concept>
+
+<Concept>
+
+## Namespace
+
+A Scaleway Messaging namespace enables systems, applications, and services to communicate with each other and exchange information. A Messaging namespace acts as a message broker, translating messages so that different formal messaging protocols can communicate with one another, even if they are written in different programming languages or implemented on different platforms. Namespaces also allow you to isolate queues and group credentials. Each namespace has a **type**, currently either [NATS](#nats) or [SQS/SNS](#sqssns) which defines the [messaging protocol](#messaging-protocol) to be used.
```
Comment (lucasscw) [inline compute/messaging/concepts.mdx:96]:
```suggestion
A Scaleway Messaging namespace enables systems, applications, and services to communicate with each other and exchange information. A Messaging namespaces allow you to isolate queues and group credentials. Each namespace has a **type**, currently either [NATS](#nats) or [SQS/SNS](#sqssns) which defines the [messaging protocol](#messaging-protocol) to be used.
```

File: compute/messaging/concepts.mdx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+</Concept>
+
+<Concept>
+
+## NATS
+
+The **N**eural **A**utonomic **T**ransport **S**ystem, or NATS, is an open-source messaging system written in the Go programming language. It is part of the Cloud Native Computing Foundation (CNCF) and has more than 40 client language implementations. The application has been designed with performance, scalability, and ease of use in mind.
```
Comment (lucasscw) [inline compute/messaging/concepts.mdx:104]:
```suggestion
The **N**eural **A**utonomic **T**ransport **S**ystem, or NATS, is an open-source messaging system written in Go. It is part of the Cloud Native Computing Foundation (CNCF) and has more than 40 client language implementations. The application has been designed with performance, scalability, and ease of use in mind.
```

**Review by lucasscw — CHANGES_REQUESTED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

File: compute/messaging/concepts.mdx (hunk 1 above)
Comment (JulienVdG) [reply compute/messaging/concepts.mdx:96]: +1 for Lucas, the 'translating messages' parts leads to thing different protocols are interoperable, while in fact they're not.

File: compute/messaging/concepts.mdx (hunk 2 above)
Comment (JulienVdG) [reply compute/messaging/concepts.mdx:104]: do we want to link to nats.io here ?

**Review by nedaehm — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**